big-int/src/operators/multiply.cpp

```diff
@@ -27,10 +27,102 @@
 
 #include <istream>
 #include <ostream>
-
-#include "../bigint.hpp"
+#include <bigint.hpp>
+#include <algorithm>
 
 namespace libbig
 {
+    largeInt largeInt::operator*(const largeInt &next_number) {
+
+        auto remove_zeroes = [] (const largeInt &x) {
+            for(int j=0; j < x.number.length(); ++j) {
+                if((x.number[j]) != '0') {
+                    return largeInt(x.number.substr(j, x.number.length()));
+                }
+            }
+            return largeInt("0");
+        };
+
+        auto append_zeroes = [] (const largeInt &x, const int factor) {
+            largeInt Answer = x;
+            for(int k = 0; k < factor; k++) {
+                Answer.number.push_back('0');
+            }
+
+            return Answer;
+        };
+
+        auto simple_multiplication = [] (const largeInt &x, const largeInt &y) {
+
+            largeInt x1 = x;
+            largeInt x2 = y;
+            largeInt adder;
+            largeInt Answer(0);
+
+            if(x1.number.length() == 0 || x2.number.length() == 0) {
+                return largeInt();
+            }
+
+            int carry { 0 }, temp, f=0;
+            for(auto i=x1.number.rbegin(); i!=x1.number.rend(); ++i) {
+                for(auto j=x2.number.rbegin(); j!=x2.number.rend(); ++j) {
+                    temp = char_int_converter(*i) * char_int_converter(*j) + carry;
+                    carry = temp/10;
+                    adder.number.push_back(char_int_converter(temp%10));
+                }
+                if(carry) {
+                    adder.number.push_back(char_int_converter(carry));
+                    carry = 0;
+                }
+                std::reverse(adder.number.begin(), adder.number.end());
+                for(int k=1; k <= f; ++k) {
+                    adder.number.push_back('0');
+                }
+                Answer = Answer + adder;
+                adder = largeInt();
+                f += 1;
+            }
+
+            if(x1.sign != x2.sign)
+                Answer.sign = false;
```

Comment:
we do have `constexpr bool`s `POSITIVE` and `NEGATIVE` in `bigint.hpp` defined globally for sign convention please use them.